### PR TITLE
add Drainer contract for DEX V1.

### PR DESCRIPTION
**Pull request by kongliangzhong**

### CODE REVIEW

File: packages/loopring_v3/contracts/aux/migrate/Drainer.sol [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+
+import "../../lib/ReentrancyGuard.sol";
+import "../../lib/Claimable.sol";
+import "../../lib/Drainable.sol";
+
```
Comment (dong77) [inline packages/loopring_v3/contracts/aux/migrate/Drainer.sol:9]: This contract enables an alternative approach of getting back assets from Loopring Exchange v1. Now you don't have to withdraw using merkle proofs (very expensive); instead, all assets will be distributed on Loopring Exchange v2 - Loopring's new zkRollup impelmentation. Please activate and unlock your address on https://exchange.loopring.io to claim your assets.

File: packages/loopring_v3/contracts/aux/migrate/Drainer.sol [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+/// @author Kongliang Zhong - <kongliang@loopring.org>
+contract Drainer is Drainable
+{
+    function canDrain(address /*drainer*/, address /*token*/)
+        public
+        override
+        view
+        returns (bool) {
+        return isOwner();
```
Comment (dong77) [inline packages/loopring_v3/contracts/aux/migrate/Drainer.sol:30]: rename isOwner to isMigrationOperator

File: packages/loopring_v3/contracts/aux/migrate/Drainer.sol [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        address recipient,
+        uint    requestedAmount
+        )
+        external
+        virtual
+        returns (uint amount);
+}
+
+/// @author Kongliang Zhong - <kongliang@loopring.org>
+contract Drainer is Drainable
```
Comment (dong77) [inline packages/loopring_v3/contracts/aux/migrate/Drainer.sol:23]: rename to MigrationToLoopringExchangeV2

File: packages/loopring_v3/contracts/aux/migrate/Drainer.sol [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        uint amount,
+        address recipient
+        )
+        external
+    {
+        require(isOwner(), "INVALID_SENDER");
+        ILoopringV3Partial(loopringV3).withdrawExchangeStake(exchangeId, recipient, amount);
+    }
+
+    function isOwner() internal view returns (bool) {
```
Comment (dong77) [inline packages/loopring_v3/contracts/aux/migrate/Drainer.sol:45]: view => pure

**Review by Brechtpd — DISMISSED** [review]

**Pull request merged**